src/pages/Home/LandingPage.jsx

```diff
@@ -0,0 +1,33 @@
+import './LandingPage.css'
+import { Link } from 'react-router-dom'
+import NavigationTop from '../../components/Navigation/navigation-top'
+import Footer from '../../components/Footer/footer'
+import LandingContent from '../../components/Landing/LandingContent'
+export default function LandingPage(){
+
+return(
+<>
+    <div>
+        <div class="ec-first-container">
+            <NavigationTop/>
+            <div class="button-category">
+            <Link to='/ProductPage'><button
```

Comment:
Suggestion: here also change class to `className`

Reply:
Done!